<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exec_window</class>
 <widget class="QDialog" name="exec_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>进程调度结果</string>
  </property>
  <widget class="QListWidget" name="list_process">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>341</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>421</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ret">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>550</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
